### PR TITLE
Warn contributors: README is generated

### PR DESCRIPTION
**Pull request by szepeviktor**
This may help future contributors.

### CODE REVIEW

File: .dev-tools/src/Readme/ReadmeCommand.php [hunk 1]
```diff
@@ -44,7 +44,8 @@ final class ReadmeCommand extends Command
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $output->write(
-            \sprintf('# %s', self::NAME) . "\n\n"
+            '<!-- This files is generated by ReadmeCommand -->' . "\n\n"
```
Comment (localheinz) [inline .dev-tools/src/Readme/ReadmeCommand.php:47]:
```suggestion
            '<!-- This file is generated by ReadmeCommand -->' . "\n\n"
```

Comment (szepeviktor) [reply .dev-tools/src/Readme/ReadmeCommand.php:47]: That was my autonomous finger!

Comment (localheinz) [reply .dev-tools/src/Readme/ReadmeCommand.php]: Probably needs a rebase on top of `master` following #918, @szepeviktor!

Comment (szepeviktor) [reply .dev-tools/src/Readme/ReadmeCommand.php]: I think README needs to be regenerated.
I let you guys do that.

Comment (kubawerlos) [reply .dev-tools/src/Readme/ReadmeCommand.php]: > I think README needs to be regenerated.

That wouldn't be enough, as `analyse` job is failing with code style violation. Also, that comment won't be displayed: https://github.com/6b7562617765726c6f73/php-cs-fixer-custom-fixers/tree/test_readme_with_html_comment

I believe the problem is NOT that `README.md` is generated, but not checking failing CI's jobs in PR. I've [added](https://github.com/kubawerlos/php-cs-fixer-custom-fixers/pull/921) message to the check with information what to do.

Comment (szepeviktor) [reply .dev-tools/src/Readme/ReadmeCommand.php]: Fixed!

**Comment by kubawerlos** [comment]
Thank you @szepeviktor and @localheinz

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**